spotbugs/src/main/java/edu/umd/cs/findbugs/classfile/engine/asm/FindBugsASM.java

```diff
@@ -19,13 +19,19 @@
 
 package edu.umd.cs.findbugs.classfile.engine.asm;
 
+import static java.lang.Boolean.parseBoolean;
+import static java.util.Objects.isNull;
+
 import org.objectweb.asm.Opcodes;
 
 /**
  * @author pugh
  */
 public class FindBugsASM {
 
-    public static final int ASM_VERSION = Opcodes.ASM6;
+    private static final String EXPERIMENTAL_PROPERTY_VALUE = System.getProperty("spotbugs.experimental");
+    private static final boolean USE_EXPERIMENTAL = isNull(EXPERIMENTAL_PROPERTY_VALUE) || parseBoolean(EXPERIMENTAL_PROPERTY_VALUE);
```

Comment:
This could be simplified by using the fallback value of `System.getProperty`, eg:

    private static final boolean USE_EXPERIMENTAL = Boolean.parseBoolean(System.getProperty("spotbugs.experimental", "true"));